gems/aws-sdk-core/spec/aws/shared_config_spec.rb

```diff
@@ -2,21 +2,20 @@
 
 module Aws
   describe SharedConfig do
-
     before(:each) do
       stub_const('ENV', {})
       allow(Dir).to receive(:home).and_raise(ArgumentError)
     end
 
-    let(:mock_credential_file) {
+    let(:mock_credential_file) do
       File.expand_path(File.join(File.dirname(__FILE__),
-        '..', 'fixtures', 'credentials', 'mock_shared_credentials'))
-    }
+                                 '..', 'fixtures', 'credentials', 'mock_shared_credentials'))
```

Comment:
These got boinked by rubocop. This happened because `File.join` wasn't on a new line. Can you expand those out?